source/geod24/LocalRest.d

```diff
@@ -1576,7 +1579,9 @@ unittest
     assert(to_node.sleepFor(40) == 42);
 
     assertThrown!Exception(to_node.sleepFor(2000));
-    Thread.sleep(2.seconds);  // need to wait for sleep() call to finish before calling .shutdown()
+    to_node.ctrl.withTimeout(Duration.zero,  // wait for the node to wake up
```

Comment:
Can we keep the timeout to 2 seconds though ?

Reply:
Wouldn't that suffer from timing issues? Maybe 3 would be better?

Reply:
Well, so would the previous version of the code, no ? But 3 works for me too, or even higher. I just prefer to have a timeout, as having an infinite wait when a test doesn't pass it not very developer-friendly.

Reply:
Indeed. It's a good catch. I've updated it.